account_asset_management_xls/wizard/wiz_account_asset_report.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2009-2018 Noviat
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import unicodedata
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class WizAccountAssetReport(models.TransientModel):
+    _name = 'wiz.account.asset.report'
+    _description = 'Financial Assets report'
+
+    # TODO: add support for all date range types
```

Comment:
Just a personal opinion, TODOs don't really make sense because they contribute to something called a https://en.wikipedia.org/wiki/Bystander_effect. It's better to create a ticket with a *wishlist* tag rather than littering the code with TODOs which in the end will not be picked up by anyone. Just an opinion, fill free to ignore this.